packages/transpiler/src/db/sqlite_database_schema.ts

```diff
@@ -52,8 +52,13 @@ export class SQLiteDatabaseSchema {
     } else if (type instanceof abaplint.BasicTypes.XStringType) {
       // it will be fine, the runtime representation of xstring is also text
       return `TEXT`;
+    } else if (type instanceof abaplint.BasicTypes.HexType) {
+      // it will be fine, the runtime representation of hex is also text
+      return `TEXT`;
     } else if (type instanceof abaplint.BasicTypes.IntegerType) {
       return `INT`;
+    } else if (type instanceof abaplint.BasicTypes.PackedType){
+      return `DECIMAL(${type.getLength()},${type.getDecimals()})`;
```

Comment:
guess this will be okay for now, but I'm not sure if it works

Reply:
ye, me neither. I don't know how to check it 🤣

Reply:
write a unit test, https://github.com/abaplint/transpiler/blob/main/test/database.ts